src/Abp.RedisCache/Runtime/Caching/Redis/AbpRedisCache.cs

```diff
@@ -97,10 +97,6 @@ public override void Set(KeyValuePair<string, object>[] pairs, TimeSpan? sliding
                                           (GetLocalizedRedisKey(p.Key), Serialize(p.Value, GetSerializableType(p.Value)))
                                          );
 
-            if (slidingExpireTime.HasValue || absoluteExpireTime.HasValue)
-            {
-                Logger.WarnFormat("{0}/{1} is not supported for Redis bulk insert of key-value pairs", nameof(slidingExpireTime), nameof(absoluteExpireTime));
-            }
```

Comment:
Why remove this warning?

Reply:
By default redis does not support multi keys set with expiry time. 

If we change the default behavior for Get() with factory. It will always trigger this warning which is unnecessary.

Reply:
We should warn because this leaks cache keys.

Alternatively, we can loop the single-key `Set` when expiry time is specified.

Reply:
Agreed, we can go with looping single key action when sliding or absolute time was given

Reply:
I found similar issue on Redis repository. See https://github.com/StackExchange/StackExchange.Redis/issues/189.

I think we continue to use `StringSet` (MSET) here and replace the warning with looping through the keys given then call `KeyExpire`